.github/workflows/industrial_ci.yml

```
@@ -17,7 +17,7 @@ jobs:
     strategy:
       matrix:
         env:
-          - { ROS_DISTRO: humble, ROS_REPO: ros }
+          - { ROS_DISTRO: jazzy, ROS_REPO: ros }
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
```

```suggestion
      - uses: actions/checkout@v4
```
変更箇所ではありませんが、Checkoutの最新バージョンが4になっているので更新するとよさそうです。
https://github.com/marketplace/actions/checkout

---

ありがとうございます。commitしました。